python/delta_sharing/reader.py

```diff
@@ -183,7 +183,7 @@ def _to_pandas(
 
         if for_cdf:
             # Add the change type col name to non cdc actions.
-            if type(action) != AddCdcFile:
```

Comment:
unrelated change?

Reply:
Build and Test /python (3.8) complains about this type comparison so I made the fix.